NBitcoin/RPC/RPCClient.cs

```diff
@@ -1183,7 +1183,7 @@ public async Task<PeerInfo[]> GetPeersInfoAsync()
 					StartingHeight = (int)peer["startingheight"],
 					SynchronizedBlocks = (int)peer["synced_blocks"],
 					SynchronizedHeaders = (int)peer["synced_headers"],
-					IsWhiteListed = (bool)peer["whitelisted"],
+					IsWhiteListed = peer["whitelisted"] != null ? (bool)peer["whitelisted"] : false,

```

Comment:
`is bool v ? v : false`

Reply:
I prefer that style too but in this case that doesn't compile and this is the style that is already used in this same method.